game/src/info/bus.rs

```diff
@@ -158,7 +159,8 @@ fn passenger_delay(ctx: &mut EventCtx, app: &App, details: &mut Details, id: Bus
         .primary
         .sim
         .get_analytics()
-        .bus_passenger_delays(app.primary.sim.time(), id);
+        .bus_passenger_delays(app.primary.sim.time(), id)
+        .collect::<std::collections::BTreeMap<_,_>>();
```

Comment:
I'm not totally consistent with it, but I prefer `use std::collections::BTreeMap` at the top